src/app/delegation_backend/src/delegation_backend/main.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+  . "delegation_backend"
+  logging "github.com/ipfs/go-log/v2"
+  "net/http"
+  "context"
+  "time"
+  "cloud.google.com/go/storage"
+  "github.com/btcsuite/btcutil/base58"
+  sheets "google.golang.org/api/sheets/v4"
+  "google.golang.org/api/option"
+)
+
+const CLOUD_BUCKET_NAME = "georgeee-o1labs-1"
```

Comment:
Nit: Does it make sense to move this to `constants.go`?

Reply:
Addressed this in my last commit